src/main/java/com/github/princesslana/eriscasper/rest/guild/CreateGuildBanRequest.java

```diff
@@ -0,0 +1,23 @@
+package com.github.princesslana.eriscasper.rest.guild;
+
+import com.github.princesslana.eriscasper.util.QueryStringBuilder;
+import org.immutables.value.Value;
+
+/**
+ * @see <a
+ *     href="https://discordapp.com/developers/docs/resources/guild#create-guild-ban-query-string-params">
+ *     https://discordapp.com/developers/docs/resources/guild#create-guild-ban-query-string-params</a>
+ */
+@Value.Immutable
+public interface CreateGuildBanRequest {
+  Integer getDeleteMessageDays();
```

Comment:
We've used Long for integer fields elsewhere

Reply:
yeah I changed it